deflate_medium.c

```diff
@@ -22,21 +22,15 @@ struct match {
 
 #define MAX_DIST2  ((1 << MAX_WBITS) - MIN_LOOKAHEAD)
 
-static int tr_tally_dist(deflate_state *s, int distance, int length) {
-    return _tr_tally(s, distance, length);
-}
-
-static int tr_tally_lit(deflate_state *s, int c) {
-    return  _tr_tally(s, 0, c);
-}
-
 static int emit_match(deflate_state *s, struct match match) {
     int flush = 0;
+    int mflush = 0;
 
     /* matches that are not long enough we need to emit as literals */
     if (match.match_length < MIN_MATCH) {
         while (match.match_length) {
-            flush += tr_tally_lit(s, s->window[match.strstart]);
+            zng_tr_tally_lit(s, s->window[match.strstart], mflush);
```

Comment:
These changes should not be part of this commit and likely not this PR.
Commits like the sweeping zng_ prefix changes should have zero effective change of code flow and performance.
The changes in this file might change both, and should get more scrutiny and not be hidden within a big commit.

Reply:
I removed them since renaming them zng_tr_tally_dist and zng_tr_tally_lit would conflict with functions that already carried the same name. I will add them back.

Reply:
Also, I don't quite understand the purpose, of them; expect with the functions, you have additional function overhead for each call.

Reply:
I can add a new PR for the removal of those extra functions after this PR, if it is necessary.